src/generators/php/index.js

```diff
@@ -304,6 +305,35 @@ class PHP extends Generator {
     return types.length > 1 ? `${types.join("|")}` : types[0];
   }
 
+  validationTypeWeight(prefixedTypeName, isExtension, model) {
+    let typeName = this.getPropNameFromFQP(prefixedTypeName);
+    let compactedTypeName = this.getCompacted(prefixedTypeName);
+    let extension = this.extensions[model.extensionPrefix];
+    switch (typeName) {
+      case "Boolean":
+        return 0;
+      case "Date":
+        return 1;
+      case "DateTime":
+        return 2;
+      case "Time":
+        return 3;
+      case "Integer":
+        return 7;
+      case "Float":
+      case "Number":
+        return 6;
+      case "Property":
+      case "Text":
+      case "URL":
+        return 5;
+      case "Duration":
+        return 4;
+      case "null":
+        return 8;
+    }
```

Comment:
Is it worth putting a default weight of 9 for classes that are not of native types? Or am I misunderstanding 🤔

Reply:
I think they'll automatically get the equivalent of 0

Reply:
![openactiv… 2020-03-12 17-14-14](https://user-images.githubusercontent.com/705668/76547343-eba2d800-6484-11ea-8f7c-4d24a9db064f.png)